MapboxNavigationUI/NavigationMapView.swift

```diff
@@ -1,10 +1,14 @@
 import Foundation
+import MapboxDirections
 
-public class NavigationMapView: MGLMapView {
+open class NavigationMapView: MGLMapView {
     
     var navigationMapDelegate: NavigationMapViewDelegate?
     
-    public override func locationManager(_ manager: CLLocationManager!, didUpdateLocations locations: [Any]!) {
+    let cap = NSValue(mglLineCap: .round)
+    let join = NSValue(mglLineJoin: .round)
+    
+    open override func locationManager(_ manager: CLLocationManager!, didUpdateLocations locations: [Any]!) {
```

Comment:
@1ec5 I think this needs to happen because of https://github.com/mapbox/mapbox-navigation-ios/pull/116/files#diff-f7f2309384eb86844377711b6c8e12caR4